app/controllers/catalog_controller.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 class CatalogController < ApplicationController
 
+  before_action(:catalog_params, only: :index)
```

Comment:
Is this still needed?

Reply:
This is needed to permit passing parameters from the Catalog Controller.